nvflare/fuel/hci/client/cli.py

```diff
@@ -118,6 +126,12 @@ def __init__(
             poc=poc,
         )
 
+        if readline and cli_history_dir and os.path.exists(cli_history_dir):
+            self.cli_history_file = os.path.join(cli_history_dir, ".cli_history")
```

Comment:
A common practice is to define a ".nvflare" folder in the user's home dir. Then you can keep anything in there.

Reply:
Got it, will change to this.